lib/infra/infra-stack.ts

```diff
@@ -55,6 +55,8 @@ export interface infraProps extends StackProps{
     readonly mlNodeStorage: number,
     readonly jvmSysPropsString?: string,
     readonly additionalConfig?: string,
+    readonly dataEc2InstanceType: InstanceType,
+    readonly mlEc2InstanceType: InstanceType,
```

Comment:
Should we enable instance type for all types of nodes including leader, etc? Keep defaults but allow option to override for others too?

Reply:
Not required as manager and client nodes not really do any heavy lifting and are only involved in book-keeping and traffic routing tasks. 
We can take it up in future if the need arises. 
@gaiksaya